scenes/quests/story_quests/shjourney/3_shjourney_intro/template_intro0.2.tscn

```diff
@@ -0,0 +1,722 @@
4ZZBVsHohXXJAcyXmgRQECDQUeWHB8RDCB3L5KyV5ow9VpWtdIAfpzLxjruAtyXIrlXPyUZatFj3UVLEhAD+oCeir0GpJmMDZGpqozG0qYjCZRUfazcis7AbwRN13lgGo6cxVij+vvdiU7PL1l91431pqo1S188KyoekhSjeFbQ9+jJKEviTqGgcNlihov01YQaVWUG9LQWmMJI1ZoZtBRNcgUxCIkzJCkkSNFzrCkpMuNkGvQ4mVlqqptkVkUUvCaI3KCr5p4KzJMla6x0lLGpJpFisUe7pKxQhPPZ6jHbQ1kYIFWVZK5Zh1MGp9hpU9JYq2+c9dE0nTcGCWKYAbVSiPIQ+kjyz7aysZ1XYrYLckfqx2jSI4KiNdxLCKLtVQm0Q0MWeI7Vf4bEjyRRB7+pPR1CzKlBQFwgVd+0azIMk2bnaVETjnrA713Fgmmc+g+FQEIrUWEw7Poaw+0ts2n1wh+cTFs+PIiCbIYRWLqu6Bxg8XFiGH0nBm6gVWstqojlaPfUQYhEsS3+HbgVxYtRATPUz8bDMTLYbKvYXe/esojXXueIdXQNszICB0GyV6smR9pA23PppUstsnQ1BJYRVKb5RKz9c4s2zhjMX65e0Ke4xNRBGTS1SYiczbcSn1HOdJZvGEa5yTBxde9R1I46TD4L4md85Re9EpMnseyAx0ufEYXN20w//u0ZOcBhhxwwCsMTiJOhfg2JGNMFkXBACyxL8kQC2DE9IwYpEYtqxdrfkd/RFlRwllMSWQUFiTXrdTKOqWwJyYoSht4wSU8Mqk5co5TmmHbHU4VFFHCJzdWsaiHzipA6owAIxTJXNJ846zXjMslAGxFnCOTdU93mf6BnEcWHBMwJdVk6cD1NK5ExVAlmct+Ucfh8GQyGgmzeUqnVsFsh6lOc8tjlnw5oea0U5Wd4XCI/QtVpZVEcwkwbePmIjYyBbC/i8jpRzU5+KBIQlbEb3JmbYC+w2Rc9cN007aySs115wgv8MTPDfbVj5m+Hk8eWI0vos61b3g3dW03KiPi7dApA01wMKN/qJGX6T6gz2YXFWxW9q1JDco8KGsRHUGZgbbbanepXWp324sJ1K9h2is7VCf3pncWKxQ5pdxbEA9gcUEwjgDUHJSM50VjhSLvMOGIDOrq7FnDnAwM6BD0CKgJ2DixaUKLVqFofX0Qcre2lu9WS85lRKSKvmY0f36yN7Ffbbfr6+l2CvGF9/XgZoQsGo0EgaGgZEYqDYiDZQUBmMLZkpmDQYBBQKPx4FA4ub88AkAnTAQMWDzsGwxwSUyV0LLFmAdMiBi05DRcpuMwUgNSNA5x4DBAiCivB8xoIAF0hoDxoAwscRRFal4YwcBQKAZUBYQBhgQCggAJWLJUTReKJUND5gkkLLFxh8grQZAMtkVNVF5zFI1NDpuQdINOIQAoQKUD4AbHwubJ0sGReWtE0QTNyLm62HGGWBBA//u0ZPCABvlxP7Vl4ABSCgg1p4gAIlIXJ7m6AAMJLCT/N4AAWQGqxxkHEoKpOs1V1KWhoMGrBoBc2GIxB4ggIKDQD1xW9dFGkvWaLdC6eggkOMZAZQcAuMXOVxcguQdgoMXOQdWjmSKJjq/////5FCcNCLl83JwuGhPl83///+tUyWiYpJGTusITqjojKhiPjjVAtVjbRlB4LFJ8SkZQiRRoTBwMSDQeZCIpo12GQkysHGQNL1TUOI7hElVCatxjKqGIOZWJ0rwsRy7rTWHcw1KpRbn4pk8ic8qZ2nQ70ph2zNYYW8LHIck2Lvv/hGo1+NL++0/bGHOVM+50lJ//+X5c/vMN/nRMzddy3cdykgPL8cdZd/Pn81+/3m/dt/H85Tw3PzkYx5qtl/8/L8///////+pat5yypel9P2kpMHlv//+D5RwPlDFMQU1FMy4xMDBVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVU6DkEQAQChSiKwAwASIJgAgPFrTkxPbBQCASOMclziRJLfM+ZmZ7V2o6q8o7JEijLHEsokSS15R2SJEVkcFBXgvQUVCGBQU2C4goqEdCCngvQU3I6EFeC9BTcjoQV4L0FNyOhBXgvQU3I6EFeC9BTcjoQV4L0FNyGFe9RUAoAQMhnCIEgNOCwJBpq1ULOFAQESUZtgIUx6rqAiVLjcZmbVTUBAVLZtmZm1U1CgKrs3GZvVeqqlGYmDBTYLiCioR0IK//u0RGCP88MmO48wwAJkxmfS5IwAQAABpAAAACAAADSAAAAEeC9BTcjoQV4L0FNyOhBXgvQU3I6EFeC9BTcjoSpMQU1FMy4xMDCqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq")
```

Comment:
Here you have embedded an MP3 file directly into the scene. Please save it as a separate `.mp3` file and reference that in the scene.